tests/test_postfix.py

```diff
@@ -0,0 +1,34 @@
+"""This module contains the tests for the postfix container, the image with postfix, sendmail and mailq pre-installed."""
+
+import time
+
+from bci_tester.data import POSTFIX_CONTAINERS
+
+CONTAINER_IMAGES = POSTFIX_CONTAINERS
+
+
+def test_postfix_status(auto_container):
+    # sleep to allow completition of postfix configuration & service to start
+    time.sleep(20)
+
+    """Check if Postfix is running inside the container."""
+    res = auto_container.connection.run_expect([0], "echo ${SERVER_HOSTNAME}")
+    assert "localhost" in res.stdout
+
+    res = auto_container.connection.run_expect([0], "postfix status")
+    assert res.rc == 0
```

Comment:
`check_output` is the equivalent of `run_expect([0], *)` which fails if the return value is non-zero.
```suggestion
    auto_container.connection.check_output("postfix status")
```